### PR TITLE
[INTERNAL] CHANGELOG.md: Add latest UI5 Tooling V3 releases

### PR DESCRIPTION
**Pull request by flovogt**
JIRA: CPOUI5FOUNDATION-846

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,45 @@
...
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-A list of unreleased changes can be found [here](https://github.com/SAP/ui5-cli/compare/v3.9.0...HEAD).
+A list of unreleased changes can be found [here](https://github.com/SAP/ui5-cli/compare/v3.11.0...HEAD).
+
+<a name="v3.11.0"></a>
+## [v3.11.0] - 2024-06-24
+### Dependency Updates
+- Bump [@ui5](https://github.com/ui5)/project from 3.9.1 to 3.9.2 [`e74c280`](https://github.com/SAP/ui5-cli/commit/e74c28018eafb9b15da56dcb45de58fbedec5ced)
+- Bump [@ui5](https://github.com/ui5)/builder from 3.4.1 to 3.5.0 [`d8bf229`](https://github.com/SAP/ui5-cli/commit/d8bf2295933998e64c4241942b2d93cb663eb068)
```
Comment (d3xter666) [inline CHANGELOG.md:11]: ES2023 support is not included in the log: https://github.com/SAP/ui5-cli/releases/tag/v3.11.0

Comment (d3xter666) [reply CHANGELOG.md:11]: I suppose it's due to formatting

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,45 @@
...
+
+
+<a name="v3.10.3"></a>
+## [v3.10.3] - 2024-05-22
+
+<a name="v3.10.2"></a>
+## [v3.10.2] - 2024-05-17
+### Dependency Updates
+- Bump [@ui5](https://github.com/ui5)/server from 3.2.0 to 3.2.1 [`f8bc6b3`](https://github.com/SAP/ui5-cli/commit/f8bc6b343c0d8cdbfcc37cbca36699b0cc9e9e36)
```
Comment (d3xter666) [inline CHANGELOG.md:20]: Bugfixes are not included: https://github.com/SAP/ui5-cli/releases/tag/v3.10.2

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,45 @@
...
+
+
+<a name="v3.10.1"></a>
+## [v3.10.1] - 2024-05-13
+### Dependency Updates
+- Bump [@ui5](https://github.com/ui5)/builder from 3.4.0 to 3.4.1 [`7c072df`](https://github.com/SAP/ui5-cli/commit/7c072df00fa205d083c68934dac42a9abec94dcc)
```
Comment (d3xter666) [inline CHANGELOG.md:26]: Bugfixes are not included: https://github.com/SAP/ui5-cli/releases/tag/v3.10.1

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,45 @@
...
+
+
+<a name="v3.10.0"></a>
+## [v3.10.0] - 2024-04-24
+### Dependency Updates
+- Bump [@ui5](https://github.com/ui5)/server from 3.1.5 to 3.2.0 [`3aae04d`](https://github.com/SAP/ui5-cli/commit/3aae04d9e60c162d42d6e5c75906cdb563bbe761)
+- Bump [@ui5](https://github.com/ui5)/builder from 3.3.1 to 3.4.0 [`023693e`](https://github.com/SAP/ui5-cli/commit/023693e4790a6c7f2ba12492fd5868f27234620e)
```
Comment (d3xter666) [inline CHANGELOG.md:33]: Some features are not included: https://github.com/SAP/ui5-cli/releases/tag/v3.10.0

**Review by d3xter666 — DISMISSED** [review]

**Review by matz3 — APPROVED** [review]
The GitHub Release also contains the notes from the individual packages while the CHANGELOG.md only contains the actual changes of ui5-cli (e.g. version bumps). AFAIK this is intended.

**Pull request merged**